src/styled/customStyles.scss

```diff
@@ -16,11 +17,200 @@
   }
 }
 
+// custom fixed header height to ensure we know the exact header height
+.custom-header-height {
+  height: 4rem;
+
+  nav {
+    height: 100%;
+  }
+}
+
+
+// add not-allowed cursor and no transform on disabled items
 * {
   &:disabled {
     cursor: not-allowed;
     &:hover {
       transform: none;
     }
   }
+}
+
+// typography *bootstrap does not come with responsive typography by default
+$custom-font-size-1: 0.875rem;
+$custom-font-size-2: 1.5rem;
+$custom-font-size-3: 2rem;
+$custom-font-size-4: 2.5rem;
+$custom-font-size-5: 3rem;
+$custom-font-size-6: 3.5rem;
+$custom-font-size-7: 4rem;
+$custom-font-size-8: 4.5rem;
```

Comment:
why 8? 
how are we using these different sizes? headings? 

https://getbootstrap.com/docs/4.3/content/typography/ suggests theres 6 h tags and 4 display headings. are these not sufficient?

Reply:
the issue is that they don't have these classes for multiple screen sizes.  If i use `display-4`, it might be too large on the smallest size device while looking fine on the largest

Reply:
for example, `display-4` looks like this on desktop:
![screen shot 2019-02-22 at 12 17 30 pm](https://user-images.githubusercontent.com/13635403/53268790-e4533580-369b-11e9-92bc-fb14f61ace79.png)

but like this on mobile:

![screen shot 2019-02-22 at 12 17 34 pm](https://user-images.githubusercontent.com/13635403/53268796-e9b08000-369b-11e9-8be3-7566313df4b9.png)

Reply:
the alternative would be to:
1) create these custom classes
2) create multiple elements and display none on certain sizes

Reply:
what if you add the responsive rules to the existing classes? `display-4` in this case

Reply:
removed the classes, will ask Hiroo to add additional media queries when needed